inductiva/resources/resources.py

```diff
@@ -81,3 +99,13 @@ def get_machine_group(name: str):
 
     except inductiva.client.ApiException as api_exception:
         raise api_exception
+
+
+def get_machine_groups():
```

Comment:
The pattern is typically the opposite to what : 
- the get method returns a list with MachineGroup objects
- the `list` calls the `get` method and does some pretty printing of its contents.

Similar to what is implemented for the tasks:
https://github.com/inductiva/inductiva/blob/20c315dad7cc180136d211ff5e179283569ad702/inductiva/tasks/methods.py#L131